inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/hive/HiveResourceOperator.java

```diff
@@ -78,6 +84,28 @@ public void createSinkResource(SinkInfo sinkInfo) {
         this.createTable(sinkInfo);
     }
 
+    private HiveSinkDTO getHiveInfo(SinkInfo sinkInfo) {
+        HiveSinkDTO hiveInfo = new HiveSinkDTO();
+        String dataNodeName = sinkInfo.getDataNodeName();
+
+        // firstly use data node info if exists
```

Comment:
Suggest using the user-defined params first, if some params were null, then get them from the data node.

Data nodes are used to store common parameters. We should support users to customize some parameters and use them preferentially.

Reply:
makes sense!